core/runtime.rs

```diff
@@ -351,6 +360,42 @@ impl JsRuntime {
       .unwrap();
   }
 
+  /// Initializes JS of provided Extensions
+  // NOTE: this will probably change when streamlining snapshot flow
```

Comment:
Not sure what this comment means, remove before landing?

Reply:
It's a note that `init_extension_js` and `init_extension_ops` should be fully merged into a `init_extensions()` method or even folded into `new` (with no `pub fn`s) as callers no longer need to handle them separately as snapshottable JsRuntimes call have full fledged ops.